apps/labs/posts/announcing-labs-annual-report-2022.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Quansight Labs 2022 Annual Report: Celebrating Growth and Sustainability'
+author: pavithra-eswaramoorthy
+published: February 7, 2023
+description: 'Presenting our first annual report! Read about our project achievements, community initiatives, and work culture.'
+category: [Community, PyData ecosystem]
+featuredImage:
+  src: /posts/hello-world-post/featured.png
+  alt: 'TODO'
+hero:
+  imageSrc: /posts/hello-world-post/hero.jpeg
+  imageAlt: 'TODO'
+---
+
+_Co-authors: Noa Tamir, Ralf Gommers, and Tania Allard._
+_Designer: Irina Fumarel_
+
+We at Quansight Labs are delighted to share our first-ever annual report with you!
+
+In these pages, we reflect on the past year and share how we're progressing towards our mission to sustain and grow community-driven open source projects and ecosystems.
+
+<!-- TODO: Add Infographic with alt-text -->
```

Comment:
HTML comments (`<!-- ... -->`) are invalid syntax inside MDX. What's worked for me is curly-brace enclosures around a Javascript-style `//` comment ([ref](https://www.wisdomgeek.com/development/web-development/react/how-to-write-comments-in-react-jsx/#:~:text=Since%20the%20contents)):

```suggestion
{
// TODO: Add Infographic with alt-text
}
```

The closing brace *has* to be on a new line, or else it gets interpreted as part of the JS comment and breaks the JSX/MDX syntax.

Reply:
Thank you!